docs/commands/csv2po.rst

```diff
@@ -14,16 +13,18 @@ your PO translations.
 If you are starting out with your own CSV files (not created by po2csv), take
 note of the assumptions of the column layout explained below.
 
+
 .. _csv2po#usage:
 
 Usage
 =====
 
-::
+.. code-block:: console
```

Comment:
Can you point to a rendering of this.  ``::`` was meant to make this fixed width.  Using `console` code block is for command line code.  This isn't that.  So I'd like to see that the rendering is actually better, not worse.  I'm happy either way but really the simple question is this: "To the reader of the docs is this more helpful", I don't see that question even asked in this PR to motivate the change.